### ISSUE TITLE
Remove hard dependency on Acala

### ISSUE THREAD

**Issue by tien** [polkawallet-io/bridge#47]
Is it possible to move to an implementation without dependencies on `@acala-network/*`?

This would be awesome for projects that doesn't use Acala sdks

**Comment by qwer951123** [comment]
Yes, we are planning to remove these dependencies or try to use monorepos instead.
